profiles/easy

```diff
@@ -710,10 +710,13 @@ org.kde.filesharing.samba.addtogroup                            auth_admin:auth_
 # KDiskMark, HDD/SSD benchmark tool (bsc#1182521)
 org.jonmagon.kdiskmark.dropcache                                auth_admin_keep:auth_admin_keep:auth_admin_keep
 
-# deeping-system-monitor pkexec actions to invoke kill, renice, systemctl as root (bsc#1181886)
+# deepin-system-monitor pkexec actions to invoke kill, renice, systemctl as root (bsc#1181886)
 com.deepin.pkexec.deepin-system-monitor.kill no:no:auth_admin_keep
 com.deepin.pkexec.deepin-system-monitor.renice no:no:auth_admin_keep
 com.deepin.pkexec.deepin-system-monitor.systemctl no:no:auth_admin_keep
+#  deepin desktop environment helpers (bsc#1070943)
+com.deepin.api.locale-helper.manage-locale no:no:yes
+com.deepin.api.locale-helper.unblock-bluetooth-devices no:no:auth_admin_keep
```

Comment:
Shouldn't this be `com.deepin.api.device.unblock-bluetooth-devices`?
https://github.com/linuxdeepin/dde-api/blob/master/device/exported_methods_auto.go
https://src.fedoraproject.org/rpms/deepin-api/blob/f33/f/deepin-api.spec